shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -83,6 +83,10 @@ public long getAccountCreationLockoutCountTTL() {
                 System.getenv().getOrDefault("ACCOUNT_CREATION_LOCKOUT_COUNT_TTL", "3600"));
     }
 
+    public long getReauthEnterEmailCountTTL() {
```

Comment:
```suggestion
    public long getReauthEnterEmailCountTTLInSeconds() {
```
Remove any doubt about the units of the ttl when someone is reading the code.